### PR TITLE
Change RSpec Book link to Amazon

### PR DESCRIPTION
**Pull request by CodingItWrong**
The PragProg no longer has a link to purchase The RSpec Book, saying it's out of print (even the e-book). Amazon still has copies available for purchase, so that seems like a better link for now.

### CODE REVIEW

**Comment by JonRowe** [comment]
Hi @CodingItWrong, the RSpec book is kinda out dated, so for now we're not going to update this link. Thanks for your input!

**Comment by CodingItWrong** [comment]
Think it makes sense to remove the link from the site then? Or better to keep it up, so people will know it's out of print?

**Comment by JonRowe** [comment]
For now we'll keep it for historical reasons, when something more up-to-date is released we'll reword this to indicate that.

**Comment by packtpartner** [comment]
Hi @JonRowe & @myronmarston , Packt has published a book on RSpec which I think could replace the existing book. Do have a look at [RSpec Essentials](https://www.packtpub.com/web-development/rspec-essentials) and do let me know if we could feature this book. I think it could prove to be a good learning resource.

**Comment by myronmarston** [comment]
> Hi @JonRowe & @myronmarston , Packt has published a book on RSpec which I think could replace the existing book. Do have a look at RSpec Essentials and do let me know if we could feature this book. I think it could prove to be a good learning resource.

I'm authoring a book for which we'll have an announcement soon.  We'll definitely be featuring it.

**Comment by packtpartner** [comment]
That's great news @myronmarston :) Can't we have 2 books listed, yours as well as the one from Packt?

**Comment by JonRowe** [comment]
Without reviewing it we couldn't really recommend an external authors one.

**Comment by packtpartner** [comment]
I agree. I can send you a review copy. Would that work @JonRowe ?

**Comment by myronmarston** [comment]
@packtpartner -- I don't see us adding a link, for a couple reasons:

- As the lead maintainer of RSpec, a book co-authored by me has "official" endorsement status that no other book has.  If I truly felt that another book was as good as my own, I might consider, but bear in mind that I'm literally the world's leading expert on RSpec (by virtue of having worked on it for so long).
- There are other books that cover RSpec.  I don't really want to go down the path of us linking to/endorsing them all.
- Personally, I've invested literally *thousands* of hours in maintaining RSpec, all w/o receiving compensation.  Now I've invested literally hundreds (or perhaps thousands; I've not kept track) of hours in co-authoring a top-notch book.  This book is my chance to earn some compensation from all the time I've spent on RSpec and the book.  I plan to use the tools at my disposal to promote my book when it becomes available.  The RSpec website is one of the main tools I have to promote my own book.  I don't plan to use those tools to promote someone else's book.  That may sound selfish, but I think it's reasonable.  Our books are, in some sense, competitors in this space, and I'm under no obligation to promote yours as en equal to mine.

**Comment by xaviershay** [comment]
Agree with @myronmarston here

**Comment by packtpartner** [comment]
Thanks for pointing out a few details. Wish you the best for your new book @myronmarston
